app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java

```diff
@@ -0,0 +1,96 @@
+package uk.co.neuralcubes.neuralates;
+
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ControlFragment extends Fragment {
+    private Spinner selectSphero, selectMuse;
+    private TextView batterySphero, batteryMuse;
+    private Button fp1, fp2, tp9, tp10;
```

Comment:
The order can be determined by an array resource of ids. It shouldn't be an issue.